src/PeopleContext.tsx

```diff
@@ -0,0 +1,44 @@
+import React, { createContext, useState } from 'react';
+import { Person } from './types';
+
+type ContextProps = {
+  peopleList: Person[];
+  setPeopleList: (list: Person[]) => void;
+  isLoading: boolean;
+  setIsLoading: (b: boolean) => void;
+  isError: boolean;
+  setIsError: (b: boolean) => void;
+};
+
+type Props = {
```

Comment:
use interface prefer. 
Types have unique features but the interface is best for scaling.